tcf_website/models/models.py

```diff
@@ -477,6 +489,106 @@ def review_count(self):
+            .distinct()
+            .annotate(
+                gpa=Coalesce(
+                    Avg(
+                        "courseinstructorgrade__average",
+                        filter=Q(courseinstructorgrade__course=course),
+                    ),
+                    Value(6) if reverse else Value(0),
+                    output_field=FloatField(),
+                ),
+                difficulty=Coalesce(
+                    Avg("review__difficulty", filter=Q(review__course=course)),
+                    Value(6) if reverse else Value(0),
+                    output_field=FloatField(),
+                ),
+                rating=Coalesce(
+                    (
+                        Avg(
+                            "review__instructor_rating",
+                            filter=Q(review__course=course),
+                        )
+                        + Avg(
+                            "review__enjoyability",
+                            filter=Q(review__course=course),
+                        )
+                        + Avg(
+                            "review__recommendability",
+                            filter=Q(review__course=course),
+                        )
+                    )
+                    / 3,
+                    # invalid value for if there are no reviews
+                    Value(6) if reverse else Value(0),
+                    output_field=FloatField(),
+                ),
+                semester_last_taught=Max(
+                    "section__semester", filter=Q(section__course=course)
+                ),
+                # ArrayAgg:
+                # https://docs.djangoproject.com/en/3.2/ref/contrib/postgres/aggregates/#arrayagg
+                section_times=ArrayAgg(
+                    Case(
+                        When(
+                            section__semester=latest_semester,
+                            then="section__section_times",
+                        ),
+                        output_field=CharField(),
+                    ),
+                    distinct=True,
+                ),
+                section_nums=ArrayAgg(
+                    Case(
+                        When(
+                            section__semester=latest_semester,
+                            then="section__sis_section_number",
+                        ),
+                        output_field=CharField(),
+                    ),
+                    distinct=True,
+                ),
+                is_teaching_this_semester=Case(
+                    When(section__semester=latest_semester, then=True),
+                    default=False,
+                    output_field=BooleanField(),
+                ),
+            )
+        )
+        return instructors
+
+    def sort_instructors_by_key(
+        self,
+        course,
+        latest_semester,
+        recent,
+        order,
+        sortby: str = "last_taught",
```

Comment:
If when grabbing the field from `sort_field` you default to "semester_last_taught", there's no point in having a default value here.